syntax/compile.go

```diff
@@ -217,6 +222,10 @@ func (pc ParseContext) compileRbinop(b ast.Branch, c ast.Children) rel.Expr {
 }
 
 func (pc ParseContext) compileIf(b ast.Branch, c ast.Children) rel.Expr {
+	log.Error(context.Background(),
```

Comment:
It's not an error. Use `log.Print`

Reply:
I think Error is better, as it can get user's attention and consider this issue seriously. I do like to use Warning, but there is not Warning API in log.